### PR TITLE
Anime-Sugoi | Update

### PR DESCRIPTION
**Pull request by Yusaki-Akito**
[+] Buttons Things UwU
[+] Removing some description

Cr. SlowLife

### CODE REVIEW

File: websites/A/Anime-Sugoi/dist/metadata.json [hunk 1]
```diff
@@ -5,13 +5,11 @@
   },
   "service": "Anime-Sugoi",
   "description": {
-    "en": "Anime-Sugoi stream your favorite anime completely for free with thai-sub and thai sound! | This website may requied vpn for access, other country will blocked!",
-    "th": "Anime-Sugoi เว็ปไซต์สตรีมมิ่งอนิเมะที่ทุกคนชื่นชอบ ฟรีเเละมีซับไทย",
-    "ga_IE": "Sruthlaigh Anime-Sugoi an anime is fearr leat go hiomlán saor in aisce le fuaim thai-sub agus Téalainnis! | Féadfaidh an láithreán gréasáin seo vpn a iarraidh ar rochtain, cuirfear bac ar thír eile!",
-    "nl": "Anime-Sugoi streamt je favoriete anime volledig gratis met thai-sub en thai sound! ​ Deze website heeft mogelijk een VPN verbinding nodig voor toegang, andere landen dab thailand worden geblokkeerd!"
```
Comment (ririxidev) [inline websites/A/Anime-Sugoi/dist/metadata.json:11]: why removed old ones?

**Review by ririxidev — CHANGES_REQUESTED** [review]
line 126: delete
all startTimestamp lines move to presenceData object
remove `path.hostname` if
use `presence.getTimestamps` and delete function inside presence
merge all consts
and remove unecessary `{}` where possible

File: websites/A/Anime-Sugoi/dist/metadata.json (hunk 1 above)
Comment (Yusaki-Akito) [reply websites/A/Anime-Sugoi/dist/metadata.json:11]: The description I think that not good enough then I remove it

File: websites/A/Anime-Sugoi/dist/metadata.json [hunk 2]
```diff
@@ -5,13 +5,11 @@
...
+    "en": "Anime-Sugoi stream your favorite anime completely for free",
+    "th": "Anime-Sugoi เว็ปไซต์สตรีมมิ่งอนิเมะที่ทุกคนชื่นชอบฟรีเเละมีซับไทย"
```
Comment (Bas950) [inline websites/A/Anime-Sugoi/dist/metadata.json:9]:
```suggestion
    "en": "Anime-Sugoi stream your favorite anime completely for free",
    "th": "Anime-Sugoi เว็ปไซต์สตรีมมิ่งอนิเมะที่ทุกคนชื่นชอบฟรีเเละมีซับไทย",
    "nl": "Anime-Sugoi streamt je favoriete anime volledig gratis"
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

File: websites/A/Anime-Sugoi/presence.ts [hunk 3]
```diff
@@ -100,6 +110,10 @@ presence.on("UpdateData", async () => {
         episode = "Movie";
         presenceData.state = info;
         presenceData.details = episode;
+        presenceData.buttons = [
+            label: "รับชมเรื่องนี้",
+            url: path.href
+    ];
```
Comment (Bas950) [inline websites/A/Anime-Sugoi/presence.ts:116]: whut? where is the object

File: websites/A/Anime-Sugoi/presence.ts [hunk 4]
```diff
@@ -52,7 +52,7 @@ presence.on("UpdateData", async () => {
   };
 
   // Presence
-  if (path.hostname == "anime-sugoi.com" || path.hostname.includes("www.")) {
+  if (path.href.includes == "anime-sugoi.com") {
```
Comment (Bas950) [inline websites/A/Anime-Sugoi/presence.ts:55]: .includes ==

never seen that before?!?

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by ririxidev — CHANGES_REQUESTED** [review]
```
line 126: delete
all startTimestamp lines move to presenceData object
remove path.hostname if
use presence.getTimestamps and delete function inside presence
merge all consts
and remove unecessary {} where possible
delete "if" statement where you check domain
```

these things still need to be fixed

**Comment by ririxidev** [comment]
No changes to my list - adding two times same function (getTimestamps) where I told you that it is already in `Presence` class. Open pull request when you will have everything ready and good.

**Pull request closed without merging**